src/DynamoSandbox/Program.cs

```diff
@@ -17,7 +17,7 @@ namespace DynamoSandbox
 {
     class Program
     {
-        private static void MakeStandaloneAndRun(string commandFilePath, ref DynamoViewModel viewModel)
+        private static void MakeStandaloneAndRun(string commandFilePath, out DynamoViewModel viewModel)
```

Comment:
This is a minor change not related to the rest of the PR but the argument should be an `out` parameter and not a `ref`